src/pages/ws/ui/customize-table-modal/columns-list/columns-list.store.ts

```diff
@@ -37,17 +33,24 @@ export class ColumnListStore {
   }
 
   public reorderColumns = (sourceIndex: number, destinationIndex?: number) => {
-    const items = Array.from(columnsStore.getExtendedColumns)
+    const items = Array.from(this.filteredColumns)
     const startItemsLength = items.length
 
     const [reorderItem] = items.splice(sourceIndex, 1)
 
-    if (destinationIndex) {
+    if (destinationIndex?.toString()) {
```

Comment:
may be it will be better: `destinationIndex != null` or `typeof destinationIndex === 'number'`

Reply:
++